raul_choque/SpaceChallenge/src/test/java/ApoloU2Test.java

```diff
@@ -0,0 +1,35 @@
+package test.java;
+
+import main.java.ApoloU2;
+import org.junit.Assert;
+
+public class ApoloU2Test {
+
+    ApoloU2 apoloU2 = new ApoloU2();
+
+    @org.junit.Test
+    public void land_ReturnFalse_ItWillExplosion() {
+        apoloU2.setChanceOfLandingCrash(1.0);
+        apoloU2.setWeight(29);
+        Assert.assertFalse(apoloU2.land());
```

Comment:
If threshold number is being randomly generated on each land() call, how can you assert that it always gonna be false? This applies to all of your other tests

Reply:
Thank you, but with my refactor I delete a test sorry.